core-feature-pack/src/main/resources/content/bin/common.ps1

```diff
@@ -70,6 +70,16 @@ if (-Not $javaOpts){
 	$javaOpts = Get-Java-Opts
 }
 
+Function Check-For-Java9+ {
+	& $JAVA --add-modules java.se -version >$null 2>&1
+
+	if ($LastExitCode -eq 0){
+		$JAVA_9_PLUS = $true
+	}
+
+	return $JAVA_9_PLUS
+}
```

Comment:
This shouldn't be tab delimited :)

Reply:
@jamezp may I ask why? I see in the whole script a lot of places delimited with tab. When I take for example next function Get-Java-Opts it is delimited in the exactly same way

Reply:
Sorry yeah. I didn't realize the script was tab delimited itself. We must not have checkstyle checking these files. No change there required.